src/stores/connectors/ethereum/utils.go

```diff
@@ -27,7 +25,7 @@ func getEIP712EncodedData(typedData *core.TypedData) (string, error) {
 }
 
 func getEIP191EncodedData(msg []byte) string {
-	return fmt.Sprintf("\x19Ethereum Signed Message\n%d%v", len(msg), hexutil.Encode(msg))
+	return fmt.Sprintf("\x19Ethereum Signed Message:\n%d%v", len(msg), string(msg))
```

Comment:
Did you find out where is documented which format message should be appended to the message?

Reply:
No, I took this implementation out of the ethers.js lib